m01_introduction/01_gallery.rst

```diff
@@ -0,0 +1,158 @@
+.. The roles  should be created at the beginning of the document. Global roles can be created in the `global.rst` file
+
+.. include:: /global.rst
+
+.. Chapter content
+
+Gallery of features
+===================
+
+.. styled-topic::
+
+  Main questions:
+    What are the main features of A+?
+
+  Topics:
+    In this section we will talk about:
+
+    * `Front page`_
+    * `Exercises`_
+    * `Services`_
+
+  Material:
+    In this chapter you won't need extra material.
+
+  Requirements:
+    There are no requirements for this chapter.
+
+  Estimated working time:
+    20min.
+
+
+
+This section is a visual index of exercises and services that have been used
+with A+.
+
+.. _front-page:
+
+Front page
+----------
+
+.. figure:: /images/gallery/aplus-front-page.png
+  :width: 80%
+  :align: center
+
+  **A+ front page Spring-2020.** In the A+ front page you can find the list of the courses (ongoing courses) uploaded in the platform`.
+
+The above image shows a screenshot of the front page of `A+ at Aalto University
+<https://plus.cs.aalto.fi/>`_ from Spring 2020. The front page shows the most recent courses. Each course has an optional logo image, name, course
+code and teaching time. All courses have a green "Aalto" tag to indicate that
+the course is for Aalto University students. The Ohjelmointistudio 2 course
+has also a blue "MOOC" tag to indicate the course is also a massive open online
+course. Sections :doc:`9.1 Settings a course on production servers
+</m09_admin/01_setup>`
+and :doc:`9.2 Course settings </m09_admin/02_settings>` describe these in detail.
+
+Exercises
+---------
+
+Point and click
```

Comment:
I wouldn't call Jsvee and JSAV point-and-click exercises myself since they are focused on program and algorithm visualizations. We have a real point-and-click exercise type in the Acos server and it is not tied to only program and algorithm visualizations.